### PR TITLE
fix: stargate: Reorganize packages and Add examples

### PR DESCRIPTION
**Pull request by youngjoon-lee**
- Added `index.ts` files, so that users can import classes by `import * from '@medibloc/panacea-js` without any sub-directory lookup.
- Updated README
- Added example codes
- Changed the version to `v2.0.0-rc.1` to test `yarn publish`.
  - We don't need a `.npmignore` file anymore because we have the `files` setting in the `package.json`.

### CODE REVIEW

File: src/did/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './util';
```
Comment (gyuguen) [inline src/did/index.ts:1]: When will this export be used?
`src/did/index.ts` is always used when import did module?

Comment (youngjoon-lee) [reply src/did/index.ts:1]: The `src/did/index.ts` is imported by `src/index.ts`. Then, end-users can import everything by just `import * from '@medibloc/panacea-js'`.
```ts
import { panaceaWalletOpts, SigningPanaceaClient, Secp256k1, DidUtil } from "@medibloc/panacea-js";
```
For more details, please see the example: https://github.com/medibloc/panacea-js/blob/3dc4decb914f346c4340bd9d7c82aabfae2364d4/docs/examples.md#creating-dids

If we don't have `index.ts` files, end-users have to know which directory they should import. For example,
```ts
import { panaceaWalletOpts, SigningPanaceaClient } from "@medibloc/panacea-js/dist/signing-panacea-client";
import { Secp256k1 } from "@medibloc/panacea-js/dist/crypto/secp256k1";
...
```

Comment (gyuguen) [reply src/did/index.ts:1]: That's what you talked about offline.
Thank you for the explanation. 😃

**Review by gyuguen — APPROVED** [review]
👍

**Pull request merged**